infrastructure-configs/demeter.json

```diff
@@ -3,14 +3,37 @@
 "substitutions": {
     "AOCS_HDFS_BAMS": "hdfs:///datasets/ovarian/ega-tim",
     "HG19_FASTA": "/hpc/users/ahujaa01/reference_genomes/hg19-reference/ucsc.hg19.fasta"
-}
+},
 
-"spark_configuration": {
-    "spark.driver.maxResultSize": "0",
-    "spark.yarn.executor.memoryOverhead": "6000",
-    "spark.executor.memory": "16g",
-    "spark.default.parallelism": "10000"
+"spark_submit_arguments": [
+    "--deploy-mode", "cluster",
+    "--master", "yarn",
+    "--executor-cores", "6",
+    "--executor-memory", "17g",
+    "--driver-memory", "10g",
 
-}
+    "--conf", "spark.driver.maxResultSize=0",
```

Comment:
Got it, thanks @ryan-williams . Can move to a config file in a future PR if desired. 

> Are you sure "--conf", "spark.executor.extraJavaOptions=-XX:+PrintGCDetails -XX:+PrintGCTimeStamps -verbose:gc" is going to get parsed properly?

Yeah, I'm at least sure of that. We're launching guacamole with the python `subprocess.check_call` function, which takes a list of arguments passed directly to the program, no shell quoting involved.